src/studio/nuxeo-studio/working-in-studio/migration-services.md

```diff
@@ -0,0 +1,120 @@
+* go back to the state your project was before triggering it anytime,
+* delete the additional configuration that was generated partially or totally.
+
+After migration is run, a detailed report is created in **Migration Services** so that you can check the results.
+
+{{#> callout type='info' heading='part of early access program'}}
+JSF UI to Web UI migration tool is available through our early access program, offering stable features you can co-create with us as they are only partially complete.<br /><br />
+
+- Not a member of it yet? [Learn about its benefits, what's planned and how to get in]({{page page="early-access-program"}}).
+<br /><br />
+- Already a member? Help us co-creating it by [telling us about your experience](https://docs.google.com/forms/d/e/1FAIpQLSdoDfpJOnFwkxHAHOk5aq5KHh9myxF6fISugnMzVmvuQSp-fw/viewform?usp=pp_url) </a>.
+{{/callout}}
+
+## What is migrated?
+
+| Migrated items | Not currently migrated |
+|----------------|-------------------|
+| Document layouts | Content views that contain query parameters or that reference a table layout |
+| Workflow tasks layouts | Translation files  |
+| Form layouts | Tabs     |
+| Content views that do not reference a table layout |     &nbsp;     |
+| User actions |        &nbsp;    |
+
+This scope is subject to change. Feel free to [let us know about your experience](https://docs.google.com/forms/d/e/1FAIpQLSdoDfpJOnFwkxHAHOk5aq5KHh9myxF6fISugnMzVmvuQSp-fw/viewform?usp=pp_url)
+
+## Processing Migration
+
+{{#> callout type='warning' heading='working during migration'}}
+Although it is possible, we recommend not to work while the migration is happening.
+
+Any configuration you would do in that time frame would not be taken into account by the migration tool, and could lead to conflicts if you decide to merge changes afterwards.
+
+{{/callout}}
+
+### Launching Migration
+
+If your project benefit from branch management: In **Source Control**, go to **Branch Management**, checkout the branch you want to migrate.
+
+
+1. In **Configuration**, go to **Migration Services**.
+1. Click on **Start Migration**.
+A pop-up appears to ensure you want to start migrating.
+1. Click **OK** to confirm.
+
+![]({{file name='migration-services-jsf-to-web-ui-screen.png'}} ?w=650,border=true)
+
+{{#> callout type='note' }}
+
+If you are using the intermediate or advanced commit modes, you need to discard or commit and push any work in progress before the migration can be launched.
+
+{{/callout}}
+
+Migration can be launched as many times as you wish: for example, to allow you to test changes made on your JSF UI configuration or benefit from new features that have been added to the migration tool.
+
+{{#> callout type='warning' }}
+
+The branch created for the migration will be deleted and created again with every migration attempt, resulting in the loss of any changes made into it that would have not been merged yet.
```

Comment:
Only happens if you DO NOT have branch management, needs to be added here!

Reply:
integrated :)